LCM/dsc/engine/ConsistencyInvoker/ConsistencyInvoker.c

```diff
@@ -87,9 +87,9 @@ char* getPythonProvider()
         strcat(result,buffer);
     }
 
-    // If python3 --version does not contain 'not found' return python3
-    if(strstr(result, "not found") == NULL) {
-	    return PYTHON3_COMMAND;
+    // If python2 --version exists
+    if(*result != '\0' && result[0] == '2') {
+	    return PYTHON2_COMMAND;
     }
 
     return PYTHON_COMMAND;
```

Comment:
When would the control come here ?
I assume it is when python is not installed, should we still return or exit ?